src/ngraph_backend_manager.cc

```diff
@@ -109,5 +112,47 @@ bool BackendManager::IsSupportedBackend(const string& backend_name) {
   return true;
 };
 
+// Backend Config functions
+BackendConfig* BackendManager::GetBackendConfig(const string& backend_name) {
+  std::lock_guard<std::mutex> lock(BackendManager::ng_backend_map_mutex_);
```

Comment:
Maybe add that comment that BackendConfig is expected to be a readonly class, hence only locked at creation, but not during later access